uber/model_checks.py

```diff
@@ -1,33 +1,35 @@
 """
```

Comment:
I think the new docstrings explains this module sufficiently.  BTW, all of the unit tests for validations are broken at the moment because I haven't had time to keep them up to date :(